bin/static-review.php

```diff
@@ -12,18 +12,23 @@
  * @see http://github.com/sjparkinson/static-review/blob/master/LICENSE
  */
 
-$included = include file_exists(__DIR__ . '/../vendor/autoload.php')
-    ? __DIR__ . '/../vendor/autoload.php'
-    : __DIR__ . '/../../../autoload.php';
+foreach (array(__DIR__ . '/../../../autoload.php', __DIR__ . '/../../vendor/autoload.php', __DIR__ . '/../vendor/autoload.php') as $file) {
+    if (file_exists($file)) {
+        define('STATIC_REVIEW_COMPOSER_INSTALL', $file);
```

Comment:
Thanks for the heads-up @keradus. See https://github.com/sjparkinson/static-review/pull/64/commits/e946a5000d6ba99455a8600bd5c48306f0787be8